example/stores/counter.ts

```diff
@@ -1,8 +1,12 @@
 import { Store } from '../../src';
 import { Stores } from '../synaptik';
 
-export default class CounterStore extends Store<CounterStore, Stores> {
-  state: { count: number } = {
+interface CounterState {
+  count: number;
+}
+
+export default class CounterStore extends Store<CounterState, Stores> {
```

Comment:
Can we also remove `this.decrement();` from inside of the `#increment`? 😅

Reply:
I'm not seeing this?

Reply:
You're right, may have been some local code that I mistook for old changes.